### PR TITLE
Feature/77887-FT-scenerio-implementation-change-update-cell

### PR DESCRIPTION
**Pull request by prashantmgupta**

### CODE REVIEW

File: tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadeGeneratedXmlFiles/367ce4a4-1d62-4f56-b359-59e178d78200.xml
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadeGeneratedXmlFiles/367ce4a4-1d62-4f56-b359-59e178d78200.xml:1]: Keep this folder empty to download xmls

**Review by Sadha1501493 — CHANGES_REQUESTED** [review]

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs [hunk 1]
```diff
@@ -14,36 +19,49 @@ public class SAPXmlHelper
...
-        static readonly List<string> AttrNotMatched = new List<string>();
+        static List<string> AttrNotMatched = new List<string>();
+        static List<string> ChangeAVCSUoS = new List<string>();
+        static List<string> ChangeENCCell = new List<string>();
         public static List<string> listFromJson = new List<string>();
         public static List<string> actionsListFromXml = new List<string>();
         public static Config config = new Config();
         private static JsonPayloadHelper jsonPayload { get; set; }
-        private static SAPXmlPayload xmlPayload { get; set; }
+        //private static SAPXmlPayload xmlPayload { get; set; }
```
[the remark this answers is not in the document]
Comment (prashantmgupta) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs]: Done

**Review by RaviK2233 — COMMENTED** [review]
@Sadha1501493  could you please have a look at the failing checks

**Review by RaviK2233 — APPROVED** [review]
Reviewed FT and all looks good

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**